frontend/app/utils/seo-utils.ts

```diff
@@ -28,3 +28,27 @@ export function useCanonicalURL(origin: string) {
 
   return url.toString();
 }
+
+/**
+ * Custom hook to generate alternate language URLs based on the current location.
+ * @param {string} origin - The origin URL. For more information on the origin URL, see {@link https://developer.mozilla.org/en-US/docs/Web/API/URL/origin}.
+ * @returns {Array<{ hreflang: string, href: string }>} An array of objects containing the hreflang and href for alternate language URLs.
```

Comment:
**Nitpick:** there's no reason to add types to the JSDoc when you're using typescript.

Reply:
Fixed